### PR TITLE
latest raw3-1 model

### PR DESCRIPTION
**Pull request by fmessmer**
This is my student's latest raw3-1 model.
It includes raw_base, raw_tower, raw_batteries,... and there are some nice meshes for the laser scanner as well. Also, the collision model looks quite good.

But it needs a critical review before being included to ipa320!

Another problem is the Universal Arm currently used. It is not displayed correctly in gazebo, but it looks good in rviz. We use the official ur5_description. Should we go back to our old own description?

Please comment!

### CODE REVIEW

File: cob_hardware_config/raw3-3/config/teleop.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+up_down: 5 #tray--up/down; tilt--front/back, here we just name up_down
+left_right: 4 #pan--left/right
+
+# component configuration
+modules: {
+  base: {
+    max_velocity: [0.3, 0.2, 0.3],
+    max_acceleration: [0.5, 0.5, 0.7]
+  },
+  tray: {
```
Comment (floweisshardt) [inline cob_hardware_config/raw3-3/config/teleop.yaml]: there should be no tray, head and sdh config

**Comment by floweisshardt** [comment]
some minor issues:
- cob_default_robot_config/raw3-1/arm_joint_configurations.yaml delete german names
- cob_default_robot_config/raw3-1/command_gui_buttons.yaml delete german names
- due to performance issues there are new configuration for the ethernet cameras, see cob3-3
- cob_hardware_config/raw3-3/config/collision_velocity_filter.launch: there should only be yaml files in hardware_config, the launch file goes into cob_bringup, see cob_bringup/tools/base_collision_observer.launch
- cob_hardware_config/raw3-3/config/teleop.yaml: why is there tray, head and sdh config?

**Comment by fmessmer** [comment]
raw3-3 is not supposed to work yet...;-)
it's under construction...

I fixed the issues about raw3-1!

File: cob_hardware_config/raw3-3/config/collision_velocity_filter.launch [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
```
Comment (floweisshardt) [inline cob_hardware_config/raw3-3/config/collision_velocity_filter.launch]: remove launch file from cob_hardware_config and use the ones on cob_bringup

**Comment by fmessmer** [comment]
Everything looks good now (for raw and for cob)!
Fixed:
- use stl, not stlb
- replace solid with robot (some stl's have been modified with meshlab -> these files start with VCG)
- no offsets anymore (origins of stl's have been moved)
- some cleaning up

Only two more things:
- Do we keep our old model of the ur5_arm? If not, the meshes and the macro inclusions for them can be removed.
- Transformation of raw_torso and camera_head are not correct yet (commented out) 
  -> I'll ask abubeck.

File: cob_hardware_config/raw3-3/config/cameras/parameters_left.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+auto_exposure:     True
+exposure_auto_max: 25000
+exposure_auto_target: 40
+exposure:          0.009
+auto_gain:         True
+gain_auto_target: 40
+gain:              1
+auto_whitebalance: True
+frame_id: head_color_camera_l_link
+stream_bytes_per_second: 55000000
```
Comment (floweisshardt) [inline cob_hardware_config/raw3-3/config/cameras/parameters_left.yaml:10]: due to perfomance issues this value is reduces for all cobs now, see https://github.com/ipa320/cob_robots/blob/master/cob_hardware_config/cob3-3/config/cameras/parameters_left.yaml

File: cob_hardware_config/raw3-3/config/cameras/parameters_right.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+auto_exposure:     True
+exposure_auto_max: 25000
+exposure_auto_target: 40
+exposure:          0.009
+auto_gain:         True
+gain_auto_target: 40
+gain:              1
+auto_whitebalance: True
+frame_id: head_color_camera_r_link
+stream_bytes_per_second: 54000000
```
Comment (floweisshardt) [inline cob_hardware_config/raw3-3/config/cameras/parameters_right.yaml:10]: see above

**Pull request merged**